formulas/mongo-express

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source "${BASH_SOURCE%/*}/../common"
+
+# Configs
+me_dock_name=dock-mongo-express
+me_dock_port=8081
+
+# Stop Container
+force_stop $me_dock_name
+
+# run Container
+run --detach \
+    --publish $me_dock_port:$me_dock_port \
+    --link dock-mongodb:mongo \
+    --name $me_dock_name \
+    knickers/mongo-express
```

Comment:
Can we reference a stable docker image tag? We have seen multiple cases where images were updated and dock formulas broken in the process. A tag would help avoiding this.